forage-android/src/main/java/com/joinforage/forage/android/network/TokenizeCardService.kt

```diff
@@ -14,11 +15,14 @@ import java.io.IOException
 
 internal class TokenizeCardService(
     private val httpUrl: HttpUrl,
-    okHttpClient: OkHttpClient
-) : NetworkService(okHttpClient) {
+    okHttpClient: OkHttpClient,
+    private val logger: Log
+) : NetworkService(okHttpClient, logger) {
     suspend fun tokenizeCard(cardNumber: String, customerId: String): ForageApiResponse<String> = try {
+        logger.i("POST request for Payment Method")
         tokenizeCardCoroutine(cardNumber, customerId)
     } catch (ex: IOException) {
+        logger.e("Failed while tokenizing PaymentMethod", ex)
```

Comment:
include `cardNumber` and `customerId` in log?

Reply:
https://github.com/teamforage/forage-android-sdk/pull/42#discussion_r1274195275